src/lab/pkg2/Lab2.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package lab.pkg2;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author JianLang LIN, ZiYing Guo
+ * @Version 3/2/15
+ */
+public class Lab2 {
+
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String[] args) {
+        Scanner Keyboard = new Scanner(System.in);//import the new Scanner
+        int cents, quarters, dimes, nickels, pennies, totalcoin, quarterscent, dimescent, nickelscents;
+        System.out.print("Enter amount of change in cents:");//Ask for user to enter how many cents
+        cents = Keyboard.nextInt();//Store the input
+        if (cents >= 0) {//Currency must be positive

```

Comment:
We didn't learn `if` statements at this point. You can assume that the change is non-negative.